pkg/pillar/zedUpload/datastore_req.go

```diff
@@ -66,6 +68,15 @@ type DronaRequest struct {
 
 	// Download counter
 	retry int
+
+	// used by Multipart upload
+	Adata     []byte
+	PartID    string
+	SasURI    string
+	Container string
+
+	// used by azure
+	Blocks []storage.Block
```

Comment:
Why do we need azure specific structure in request structure? Can't we pass a string array of ids and prepare azure specific structure inside azure utils before putting a blocklist?

Reply:
make sense, will change accordingly.